stream-chat-android-compose/src/main/java/io/getstream/chat/android/compose/ui/messages/header/MessageListHeader.kt

```diff
@@ -149,7 +160,7 @@ public fun DefaultMessageListHeaderCenterContent(
 
     Column(
         modifier = modifier
-            .fillMaxHeight()
+            .height(IntrinsicSize.Max)
```

Comment:
This change allows whatever Composable wraps this component to use `Modifier.wrapContentHeight()` properly. 

`Modifier.fillMaxHeight()` just forces filling the maximum height even if the parent Composable uses `Modifier.wrapContentHeight()`. It's still possible to force wanted behavior in either case by using  `Modifier.height(IntrinsicSize.Min)` but "ignoring"  `Modifier.wrapContentHeight()` is bad behavior.